recipe/meta.yaml

```diff
@@ -26,7 +26,9 @@ requirements:
     - setuptools_scm
   run:
     - python
-    - 'adc[channel=scimma]'
+    - adc >=0.0.3
```

Comment:
is there a reason why the channel specifier was removed?

Reply:
Specifying both the channel and the version did not work during my tests (specifying the version requires quotes, which didn't appear to work when also specifying the version).

And it turns out that specifying the channel here is redundant since the `scimma` channel is already added in the [conda build/upload workflow](https://github.com/scimma/client_library/blob/ee6165e3274635ecda90573d44fb74170ae465cf/.github/workflows/upload_package.yml#L72), so `conda build` finds `adc` without specifying the channel here in the recipe.

Reply:
Makes sense, thanks for the clarification.